org.opent2t.onboarding.contosohub/js/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "opent2t-onboarding-org-opent2t-onboarding-contosohub",
+  "version": "0.0.2",
+  "description": "Onboarding for ContosoThings",
+  "license": "MIT",
+  "main": "thingOnboarding.js",
+  "files": [
+    "thingOnboarding.js",
+    "common.js",
```

Comment:
I removed all common.js when I did the authUpdate, but I must have missed the package.json (apparently, though I swear I made the change...).  It doesn't seem to break pack-translator/npm pack/npm publish though.  It should be removed from here, and as a followup, should be removed from everywhere.

---
In reply to: [103514688](https://github.com/openT2T/onboarding/pull/32#discussion_r103514688) [](ancestors = 103514688)